plugins/extmon.c

```diff
@@ -363,14 +365,34 @@ void plugin_extmon_load_config(vscf_data_t* config, const unsigned num_threads V
         helper_path = gdnsd_resolve_path_libexec("gdnsd_extmon_helper", NULL);
 }
 
+#define SVC_OPT_UINT(_hash, _typnam, _loc, _min, _max) \
+    do { \
+        vscf_data_t* _data = vscf_hash_get_data_byconstkey(_hash, #_loc, true); \
+        if(_data) { \
+            unsigned long _val; \
+            if(!vscf_is_simple(_data) \
+            || !vscf_simple_get_as_ulong(_data, &_val)) \
+                log_fatal("plugin_http_status: Service type '%s': option '%s': Value must be a positive integer", _typnam, #_loc); \
```

Comment:
paste from another plugin left its name in the error messages (my fault really, the vscf stuff needs better abstractions)